Actor

```diff
@@ -9,16 +9,18 @@ public class Actor {
 	 * clearance = 2, nurse
 	 * clearance = 3, doctor
 	 * clearance = 4, CEO
+	 * clearance = 5, timer
 	 */
 	//empty constructor lol
 	public Actor()
 	{
 		
 	}
 	
-	public Actor(String name, int clearance)
+	public Actor(String name, int clearance,String password)
 	{
 		this.name = name;
 		this.clearance = clearance;
+		this.password = password;
 	}
```

Comment:
Add an ID number to all actors. Patients need patient ID numbers and all hospital people need an ID to login.